setup/pi/setup-teslausb

```diff
@@ -841,6 +841,17 @@ then
   echo "source /root/bin/setup-teslausb" >> /root/.bashrc
 fi
 
+# UX courtesy reminders
+if ! grep -q remount /root/.bashrc
+then
+  echo "echo \"Use bin/remountfs_rw to allow changes.\"" >> /root/.bashrc
```

Comment:
I would add or replace this with "Run 'bin/setup-teslausb upgrade' to update teslausb to the latest version." since "how do I update?" is a common question.